mon-pix/tests/unit/utils/session-storage-entry-test.js

```diff
@@ -0,0 +1,76 @@
+import { SessionStorageEntry } from 'mon-pix/utils/session-storage-entry';
+import { module, test } from 'qunit';
+
+module('Unit | Utilities | SessionStorageEntry', function () {
+  test('sets and gets a string value', function (assert) {
+    // given
+    const key = new SessionStorageEntry('testKey');
+    const value = 'testValue';
+
+    // when
+    key.set(value);
+
+    // then
+    assert.strictEqual(key.get(), value);
```

Comment:
Modification de `key`→`entry` dans tout le reste des tests de ce fichier : 
```suggestion
    const entry = new SessionStorageEntry('testKey');
    const value = 'testValue';

    // when
    entry.set(value);

    // then
    assert.strictEqual(entry.get(), value);
```